src/toil/jobStores/awsJobStore.py

```diff
@@ -865,3 +981,66 @@ def parse_bool(s):
         return False
     else:
         raise ValueError(s)
+
+class File(object):
```

Comment:
Rename this to `FileInfo`. And rename every variable called `file` to `fileInfo` if it holds a reference to a `FileInfo` instance. That way we don't hide the `file` built-in.